src/compiler/sys.ts

```diff
@@ -1319,6 +1319,16 @@ namespace ts {
             };
             return nodeSystem;
 
+            /**
+             * `throwIfNoEntry` was added so recently that it's not in the node types.
+             * This helper encapsulates the mitigating usage of `any`.
+             * See https://github.com/nodejs/node/pull/33716
+             */
+            function statSync(path: string): import("fs").Stats | undefined {
+                // throwIfNoEntry will be ignored by older versions of node
+                return (_fs as any).statSync(path, { throwIfNoEntry: false });

```

Comment:
Instead of asserting on the type of `_fs`, maybe assert on the type of the argument.

Reply:
I'm changing the return type too.  Would you prefer to have to type assertions or leave it as is?

Reply:
Nah, it's fine.